tests/contract_testcases/solana/annotations/bad_accounts.sol

```diff
@@ -75,7 +75,7 @@ contract OldAnnotationSyntax {
 // error: 39:12-29: 'SysvarInstruction' is a reserved account name
 // error: 47:12-18: account 'solang' already defined
 // 	note 46:5-19: previous definition
-// error: 58:11-18: '@seed' annotation on top of a constructor only accepts literals
-// error: 59:12-20: '@space' annotation on top of a constructor only accepts literals
-// error: 60:11-18: '@bump' annotation on top of a constructor only accepts literals
+// error: 58:11-18: expression not allowed in constant number expression
+// error: 59:12-20: expression not allowed in constant number expression
+// error: 60:11-18: expression not allowed in constant number expression
```

Comment:
this is worse

Reply:
That is the error message that stems from `eval_constant_expression`, when it does not encounter a constant expression. I thought the code would be simplified if I used the same error we already have, but I can revert it back to `annotation only supports literals`.